src/daemon/docker_exec.sh

```diff
@@ -76,6 +76,9 @@ function teardown {
   do_stayalive
 
   echo "teardown: Bye Bye, container will die with return code $exit_code"
+  if [[ $exit_code -ne "0" ]]; then
+    echo "teardown: if you want me to stayalive, next time run me with '-e DEBUG=stayalive'"
```

Comment:
If you want me not to die and offer a shell to debug this situation, next time ....